main/.vitepress/config.mjs

```diff
@@ -127,23 +127,37 @@ export default defineConfig({
           items: [
             {
               text: 'Key Concepts and APIs',
-              link: '/guides/orchestration/getting-started/key-concepts',
+              link: '/guides/orchestration/key-concepts',
             },
             {
               text: 'Contract Walkthroughs',
-              link: '/guides/orchestration/getting-started/contract-walkthroughs',
+              link: '/guides/orchestration/contract-walkthrough',
               items: [
                 {
                   text: 'Send Anywhere Example',
-                  link: '/guides/orchestration/getting-started/contract-walkthrough/send-anywhere',
+                  link: '/guides/orchestration/contract-walkthrough/send-anywhere',
                 },
                 {
-                  text: 'Cross-Chain Swap Example',
-                  link: '/guides/orchestration/getting-started/contract-walkthrough/cross-chain-swap',
+                  text: 'Cross-Chain Unbond Example',
+                  link: '/guides/orchestration/contract-walkthrough/cross-chain-unbond',
+                }
+              ]
+            },
+            {
+              text: 'Example Orchestration DApp',
```

Comment:
Ideally, like we discussed in today's sync, I would like this to be after key concepts but it doesn't have to be done in this PR. We would also need to rethink what to call that section because then `Example Orchestration Dapp` wouldn't work imo